crates/ironrdp-rdpsnd/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+use ironrdp_pdu::{gcc::ChannelName, PduEncode, PduResult};
+use ironrdp_svc::{AsAny, CompressionCondition, StaticVirtualChannel};
+use std::any::Any;
+
+/// We currently don't implement any of rdpsnd, however it's required
+/// for rdpdr to work: [\[MS-RDPEFS\] Appendix A<1>](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpefs/fd28bfd9-dae2-4a78-abe1-b4efa208b7aa#Appendix_A_1)
```

Comment:
Nit: markdown tip, you can [use the reference-style syntax](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#link-all-the-things).

```suggestion
/// for rdpdr to work: [\[MS-RDPEFS\] Appendix A<1>]
///
/// [\[MS-RDPEFS\] Appendix A<1>]: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpefs/fd28bfd9-dae2-4a78-abe1-b4efa208b7aa#Appendix_A_1)
```

Reply:
a3babe8342e4c869c858d8fe64f6b2416e8f8c59